Update Debian installation instructions in README.md

mattock opened the pull request:
Signed-off-by: Samuli Seppänen <samuli@openvpn.net>

domcleal commented on README.md:
```diff
@@ -54,6 +55,21 @@ EL users must first [configure EPEL](http://fedoraproject.org/wiki/EPEL/FAQ#How_
+	make build
+	sudo dpkg -i freight_0.3.5-1_all.deb
+
+See FPM's documentation for details about the command-line options.
```
Not relevant?

mmoll commented on README.md:
```diff
@@ -43,10 +43,11 @@ Serve `/var/cache/freight` via your favorite web server and install it as an APT
-	sudo wget -O /etc/apt/trusted.gpg.d/rcrowley.gpg http://packages.rcrowley.org/keyring.gpg
-	sudo apt-get update
-	sudo apt-get -y install freight
+	sudo -s
```
that line can be ommited, IMHO

mmoll commented on README.md:
```diff
@@ -54,6 +55,21 @@ EL users must first [configure EPEL](http://fedoraproject.org/wiki/EPEL/FAQ#How_
+
+	cd freight
+	make build
+	sudo dpkg -i freight_0.3.5-1_all.deb
```
-2 then also here

mattock commented:
I didn't have any strong opionions regarding the "Install from custom-made package" section, so I removed it as requested. It was mostly meant for people interested in repackaging freight and as a reminder for ourselves.

I also changed the apt instructions to conform to the general style of README.md by adding a "sudo" in front of every command. That said, "sudo" is not used consistently through README.md - for example:
```
freight-add ...
freight-cache ...
yum -y install freight
```
We might want to prepend these with "sudo" also, or mention that typically you'd run these commands as root. If you want, I can fix this in this PR.

domcleal commented:
> I didn't have any strong opionions regarding the "Install from custom-made package" section, so I removed it as requested. It was mostly meant for people interested in repackaging freight and as a reminder for ourselves.

Oh sorry, we might have misunderstood each other.  I was just meaning the line about fpm command line options that I commented on, not the whole section - since when running `make build` there's no way to pass those command line options in.

mattock commented:
Oh, I see. The part you commented on was a leftover indeed. Shall I add back the "Install from custom package" section?

domcleal commented on README.md:
```diff
@@ -43,10 +43,19 @@ Serve `/var/cache/freight` via your favorite web server and install it as an APT
-	sudo apt-get -y install freight
+	wget -O - https://swupdate.openvpn.net/repos/repo-public.gpg|sudo apt-key add -
+	echo "deb http://build.openvpn.net/freight_team $(lsb_release -sc) main" | sudo tee  /etc/apt/sources.list.d/freight.list
+	apt-get update
```
keep sudo on these two lines?

mattock commented:
@domcleal: sudo is only needed after the pipe.

domcleal commented:
If the instructions are meant to be executed as a non-root user as a whole then they'd be needed on apt-get calls too.

mattock commented:
You're correct, I noticed that right after sending the comment :). Fixing.

domcleal commented:
:+1:

mmoll commented:
merged, thanks @mattock

mattock commented:
No problem! This PR underlined the fact why mandatory review is good. It was surprisingly hard to get even a trivial commit like this just right.
